airgun/views/product.py

```diff
@@ -261,3 +261,17 @@ class ProductAdvancedSync(BaseLoggedInView):
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(self.title, exception=False) is not None
+
+
+
+class ProductVerifyContentChecksum(BaseLoggedInView):
+    """Represents Advanced Sync page for Products."""
+
+    #title = Text("//h4[text()='Verify Content Checksum']")
+    title = Text("//h4[normalize-space(.)='Verify Content Checksum']")
+    task = Text("//a[normalize-space(.)='Click to monitor task progress.']")
+    #sync = Text('//button[@ng-click="ok()"]')
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(self.title, exception=False) is not None
```

Comment:
```suggestion
class ProductVerifyContentChecksum(BaseLoggedInView):
    """Represents Advanced Sync page for Products."""

    task_alert = Text("//a[normalize-space(.)='Click to monitor task progress.']")

    @property
    def is_displayed(self):
        return self.browser.wait_for_element(self.title, exception=False) is not None 
```
This is just targeting the button in the alert. I'm guessing someone has already written a view for this that we can steal.